src/ledger/rocksdb.zig

```diff
@@ -98,10 +112,15 @@ pub fn RocksDB(comptime column_families: []const ColumnFamily) type {
             );
         }
 
-        pub fn get(self: *Self, comptime cf: ColumnFamily, key: cf.Key) anyerror!?cf.Value {
+        pub fn get(
```

Comment:
```suggestion
        pub fn get(
            self: *Self,
            allocator: Allocator,
            comptime cf: ColumnFamily,
            key: cf.Key,
        ) anyerror!?cf.Value {
```

Reply:
I'm fine with changing this, but just want to explain my reasoning first.

I prefer the column family to come first because I think it communicates the meaning of the code more clearly.

When written like this:
```zig
db.get(schema.slot_meta, slot, self.allocator);
```
It's almost like the column family is part of the method name, which is reasonable since it determines the types of the parameters and return of the function.
```zig
db.getSlotMeta(slot, self.allocator);
```

The allocator argument tells you less about the meaning of the code. It's more of a minor detail, which is why it feels cleaner to me at the end.

Reply:
f59c2103b90928234a562f530f0164ab957886e0